symphony-bdk-spring/symphony-bdk-app-spring-boot-starter/src/main/java/com/symphony/bdk/app/spring/auth/service/JwtService.java

```diff
@@ -0,0 +1,32 @@
+package com.symphony.bdk.app.spring.auth.service;
+
+import com.symphony.bdk.app.spring.auth.model.BdkAppErrorCode;
+import com.symphony.bdk.app.spring.auth.model.UserId;
+import com.symphony.bdk.app.spring.auth.model.exception.AppAuthException;
+import com.symphony.bdk.core.auth.ExtensionAppAuthenticator;
+import com.symphony.bdk.core.auth.exception.AuthInitializationException;
+import com.symphony.bdk.core.auth.jwt.UserClaim;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class JwtService {
```

Comment:
and perhaps move       AppAuthSession authSession = extensionAppAuthenticator.authenticateExtensionApp(token);
 to this CircleOfTrustService so that the controller uses this service only